### PR TITLE
Update mailing lists to LF AI & Data

### PR DESCRIPTION
**Pull request by porunov**
Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,28 +64,39 @@ tools:
...
     subscribe/unsubscribe with an arbitrary email address, send an email to:
 
-    * janusgraph-dev+subscribe (at) googlegroups.com
-    * janusgraph-dev+unsubscribe (at) googlegroups.com
+    * janusgraph-dev+subscribe (at) lists.lfaidata.foundation
+    * janusgraph-dev+unsubscribe (at) lists.lfaidata.foundation
 
+  * **janusgraph-announce (at) lists.lfaidata.foundation**
+    ([archives](https://lists.lfaidata.foundation/g/janusgraph-announce/topics))
+    for internal implementation of JanusGraph itself
```
Comment (mbrukman) [inline README.md:91]: Please update this description; maybe this should also be listed first?

File: README.md [hunk 2]
```diff
@@ -64,28 +64,39 @@ tools:
...
+
+    To join with a LF AI & Data account, use the [web
+    UI](https://lists.lfaidata.foundation/g/janusgraph-announce/join); to
+    subscribe/unsubscribe with an arbitrary email address, send an email to:
+
+      * janusgraph-dev+subscribe (at) lists.lfaidata.foundation
```
Comment (mbrukman) [inline README.md:97]: Please update the list name here and on the next line.

**Review by mbrukman — CHANGES_REQUESTED** [review]
Thanks for updating this!

**Comment by mbrukman** [comment]
You probably also want to add `[skip ci]` to the commit first line to avoid extra CI builds, since `README.md` is not built even as part of docs.

**Comment by porunov** [comment]
`[skip ci]` is obsolete science we moved to GitHub Actions. Instead of relying on commit message we now use specific GH Actions features to mark specific commits as documentation only update or not

File: README.md (hunk 1 above)
Comment (porunov) [reply README.md:91]: Thanks! Missed that. Will do in a minute

File: README.md [hunk 3]
```diff
@@ -64,28 +64,39 @@ tools:
...
 
-    * janusgraph-users+subscribe (at) googlegroups.com
-    * janusgraph-users+unsubscribe (at) googlegroups.com
+    * janusgraph-users+subscribe (at) lists.lfaidata.foundation
+    * janusgraph-users+unsubscribe (at) lists.lfaidata.foundation
 
-  * **janusgraph-dev (at) googlegroups.com**
-    ([archives](https://groups.google.com/group/janusgraph-dev))
+  * **janusgraph-dev (at) lists.lfaidata.foundation**
+    ([archives](https://groups.google.com/group/janusgraph-dev/topics))
```
Comment (mbrukman) [inline README.md:79]: This is pointing to Google Groups instead of LF AI & Data; please update the URL.

**Review by mbrukman — CHANGES_REQUESTED** [review]

**Comment by porunov** [comment]
@mbrukman Thank you! Updated PR per your commits!

File: README.md [hunk 4]
```diff
@@ -64,28 +64,39 @@ tools:
...
 
+  * **janusgraph-announce (at) lists.lfaidata.foundation**
+    ([archives](https://lists.lfaidata.foundation/g/janusgraph-announce/topics))
+    for new releases and news announcements
+
+    To join with a LF AI & Data account, use the [web
+    UI](https://lists.lfaidata.foundation/g/janusgraph-announce/join); to
+    subscribe/unsubscribe with an arbitrary email address, send an email to:
+
+      * janusgraph-announce+subscribe (at) lists.lfaidata.foundation
```
Comment (mbrukman) [inline README.md:97]: (Nit picking) this is indented more than the paragraph before it, which differs from the blocks above. But appears to look the same in the rendered Markdown, so up to you if you want to change it or not.

**Review by mbrukman — APPROVED** [review]
LGTM

**Comment by farodin91** [comment]
@porunov Could you also update the issue templates? Or we merge the PR #2252. I will update the tomorrow.

File: README.md (hunk 4 above)
Comment (porunov) [reply README.md]: Thanks for the catch! Aligned it to be the same as above blocks

**Comment by porunov** [comment]
@farodin91 You are redesigning the issue templates in #2252. So I think it makes sense to update it in your PR

**Comment by farodin91** [comment]
@porunov Okay, I will update PR tomorrow morning.

**Comment by porunov** [comment]
The related PR to update JanusGraph main page with new mailing lists: https://github.com/JanusGraph/janusgraph.org/pull/81

**Pull request merged**